solr/config/configsets/all_collection_oag56_v1/managed-schema

```diff
@@ -506,6 +509,7 @@
   <field name="horizontal" type="boolean" uninvertible="true" docValues="true" indexed="true" stored="true"/>
   <field name="id" type="string" termPositions="true" termVectors="true" termOffsets="true" required="true" useDocValuesAsStored="true"/>
   <field name="internal" type="boolean" uninvertible="true" docValues="true" indexed="true" stored="true"/>
+  <field name="journal" type="text_general"/>
```

Comment:
Have you added this field by yourself? Is it a necessary field? It seems not. If that is the case, delete it.

Reply:
It was already in the schema which I copied. I removed it.